src/tablet/tablet_impl.h

```diff
@@ -424,6 +425,8 @@ class TabletImpl : public ::openmldb::api::TabletServer {
     std::string notify_path_;
     std::string sp_root_path_;
     ::openmldb::type::StartupMode startup_mode_;
+
+    std::unique_ptr<::openmldb::sdk::SQLClusterRouter> sr_ = nullptr;
```

Comment:
It's better to use distribute iterator(not implemented in cluster mode now) to get all data.

Reply:
Yes. It would be nice if we have DistributedIterator. But I think it is not trivial to implement?

Any suggestion? Maybe we make an issue and optimize in a later phase.

Reply:
ok, we can optimize if distributed iterator has been implemented